libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/LedgerTypes/Specs.hs

```diff
@@ -434,7 +441,17 @@ certStateSpec acct epoch = constrained $ \ [var|certState|] ->
   match certState $ \ [var|vState|] [var|pState|] [var|dState|] ->
     [ satisfies vState (vstateSpec epoch)
     , satisfies pState (pstateSpec epoch)
-    , reify pState psStakePoolParams (\ [var|poolreg|] -> satisfies dState (dstateSpec acct poolreg))
+    , reify
+        pState
+        psStakePoolParams
+        ( \ [var|poolreg|] ->
+            reify
+              vState
+              vsDReps
+              ( \ [var|dreps|] ->
+                  satisfies dState (dstateSpec acct poolreg dreps)
+              )
+        )
```

Comment:
I'm not sure why this didn't use `match` instead of `reify`. This approach makes it impossible to test a value against this spec (but also that's fine for now, because we only care about generation of values for conformance testing)

Reply:
I guess it becomes too long when using `match`, but you can use `sel` instead to pick the field you actually care about

Reply:
I'm not sure either and just followed the pattern established by `reify pState...`. Won't address this now, we can clean up later.